dev/storage.h

```diff
@@ -835,20 +909,20 @@ namespace sqlite_orm {
             sync_schema_result schema_status(const storage_impl<table_t<T, WithoutRowId, Cs...>, Tss...>& tImpl,
                                              sqlite3* db,
                                              bool preserve) {
-                auto dbTableInfo = this->pragma.table_info(tImpl.table.name);
+                auto dbTableInfo = this->pragma.table_xinfo(tImpl.table.name);  // should include generated
```

Comment:
such comments better place as comments inside PR not inside the code cause they are very specific

Reply:
where in PR? I removed it